src/main/java/ca/ciccc/Assignment1.java

```diff
@@ -127,7 +176,43 @@ public static String powers(int number) {
      *          Min integer: 5"
      */
     public static String arithmetic(int a, int b) {
-
-        return "";
+        int sum = a + b;
+        int dif = a - b;
+        int pro = a * b;
+        double ave = sum / 2.00;
+
+        DecimalFormat df = new DecimalFormat();
+        df.setMinimumFractionDigits(2);
+        df.setRoundingMode(RoundingMode.DOWN);
+
+        String aves = df.format(ave);
+
+
+
+        if(a > b){
```

Comment:
"return ..." is duplicated. I think you can put it out after if cause and declare some variable before if.

Reply:
Yes, you should refactor your code to avoid repeating code